src/main/java/org/mule/extension/ftp/internal/ftp/command/FtpCommand.java

```diff
@@ -73,7 +73,7 @@ public FtpFileAttributes getFile(String filePath) {
    */
   @Override
   protected boolean exists(Path path) {
-    return getFile(path.toString()) != null;
+    return getBasePath(fileSystem).equals(path) || getFile(path.toString()) != null;
```

Comment:
this is good but not enough... the user could give you a path like `../../` which is valid. On top of what you added,   you should explicitly consider the case of path being `/`